install.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+git submodule update
```

Comment:
The reason I never had the submodule update in the install script is because users would have different versions of the RBGMatrix library on their systems, making things a bit of a pain to help debug. Keeping that submodule version pinned removes one extra variable.

Any particular reason we need to update the submodule as part of the feedparser fix?

Reply:
I believe `dev` hasn't been updated since the last round of changes a couple months ago. The only change on my commit was `feedparser`: https://github.com/MLB-LED-Scoreboard/mlb-led-scoreboard/pull/301/commits/7b6da126fe16f8288f1e5b18439177ae4f0ef2bb

Reply:
I think this change is still coming from your fork, though. I don't see `git submodule update` in master or dev on this repo.

The version bump is definitely from dev, but this change isn't.

Reply:
Hmm. I see that now. I pushed the commit from my Pi I was testing on... Looks like something got out of sync somewhere. I'll close this until I can get a clean PR up.